packages/arb-token-bridge-ui/src/util/NumberUtils.ts

```diff
@@ -9,23 +9,96 @@ export function formatUSD(value: number) {
   return `$${formattedValue}`
 }
 
-export function formatNumber(value: number, maximumFractionDigits: number = 6) {
-  return value.toLocaleString(undefined, {
-    maximumFractionDigits
-  })
+export enum MaximumFractionDigits {
+  None = 0,
+  Short = 1,
+  Compact = 3,
+  Standard = 4,
+  Long = 5
 }
 
-export function formatBigNumber(
-  value: BigNumber,
-  decimals: number = 18,
-  maximumFractionDigits?: number
-) {
-  if (value.isZero()) {
-    return '0'
+/**
+ * Parse number according to english formatting.
+ *
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat
+ *
+ * Should not be used directly, use formatAmount instead
+ */
+const formatNumber = (
```

Comment:
In this case, to avoid accidental usage, can we name this function with an internal notation. ie. `const _formatNumber = (`

Reply:
The function is not exported from the file so that should be enough?